docs/releases/5.2.md

```diff
@@ -50,3 +50,24 @@ depth: 1
 ### `ModelViewSet` automatically registers the model to the reference index
 
 Models that are registered with a `ModelViewSet` now have reference index tracking enabled by default. This means that you no longer need to call `ReferenceIndex.register_model()` in your app's `ready()` method for such models. If this is undesired, you can disable it by setting {attr}`~wagtail.admin.viewsets.model.ModelViewSet.add_to_reference_index` to `False` on the `ModelViewSet` subclass. For more details, see [](managing_the_reference_index).
+
+### Breadcrumbs now use different data attributes and events
+
+The undocumented JavaScript implementation for the header breadcrumbs component has been migrated to a Stimulus controller, this may impact custom header implementations that relied on the previous `initCollapsibleBreadcrumbs` approach.
+
+#### Data attributes
+
+| Old                       | New                                                                                                 |
+| ------------------------- | --------------------------------------------------------------------------------------------------- |
+| `data-breadcrumb-next`    | `data-controller="w-breadcrumb"`                                                                    |
+| `data-toggle-breadcrumbs` | `data-w-breadcrumb-target="toggle" data-action="w-breadcrumb#toggle mouseenter->w-breadcrumb#peek"` |
+| `data-breadcrumb-item`    | `data-w-breadcrumb-target="content"`                                                                |
+
+Note that the root DOM element also includes a set of additional data attributes to function as the breadcrumbs, these are best used via the existing shared template or shared header includes.
+
+#### Events
+
+| Old                     | New                              |
+| ----------------------- | -------------------------------- |
+| `'w-breadcrumb:opened'` | `'wagtail:breadcrumbs-expand'`   |
+| `'w-breadcrumb:closed'` | `'wagtail:breadcrumbs-collapse'` |
```

Comment:
looking at https://github.com/wagtail/wagtail/pull/10793/files#diff-d5bb1562a994211c12dcdeee92f076598ad97f5ab290978072bdf676be3f01beR60 the columns are swapped here?

Reply:
Good call out. I'll fix this up.